webpack.config.js

```diff
@@ -101,7 +101,12 @@ const GutenbergBlocksConfig = {
 			{
 				test: /\.jsx?$/,
 				exclude: /node_modules/,
-				loader: 'babel-loader?cacheDirectory',
+				use: {
+					loader: 'babel-loader?cacheDirectory',
+					options: {
+						presets: [ '@wordpress/default' ],
```

Comment:
What is the difference between `@wordpress/default` and `@wordpress/babel-preset-default` (which was used in `babel.config.js` before)?

Reply:
This is what is documented on the package here https://www.npmjs.com/package/@wordpress/babel-preset-default